lib/cisco_node_utils/bgp.rb

```diff
@@ -56,8 +56,10 @@ def self.routers
 
       hash_final = {}
       asnum = asnum.to_i unless /\d+.\d+/.match(asnum)
+      # rubocop:disable Style/AlignHash
       hash_tmp = { asnum =>
         { 'default' => RouterBgp.new(asnum, 'default', false) } }
+      # rubocop:enable Style/AlignHash
```

Comment:
Rubocop's autocorrect is:

```
hash_tmp = { asnum =>
                       { 'default' => RouterBgp.new(asnum, 'default', false) } }
```

And I couldn't find a cleaner solution that Rubocop approved of.